src/App.tsx

```diff
@@ -3,29 +3,65 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { Todo } from './react-app-env';
+import { getTodos } from './api';
 
 interface State {
-  selectedUserId: number;
+  selectedUserId: number,
+  todos: Todo[],
+  errorMessage: string,
 }
 
 class App extends React.Component<{}, State> {
   state: State = {
     selectedUserId: 0,
+    todos: [],
+    errorMessage: '',
+  };
+
+  async componentDidMount() {
+    try {
+      const todos = await getTodos();
+
+      this.setState({ todos });
+    } catch {
+      this.setState({ errorMessage: 'List is not found' });
+    }
+  }
+
+  selectUser = (userId: number) => {
+    if (this.state.selectedUserId === userId) {
+      return;
+    }
+
+    this.setState({ selectedUserId: userId });
+  };
+
+  clearUser = () => {
+    this.setState({ selectedUserId: 0 });
   };
 
   render() {
-    const { selectedUserId } = this.state;
+    const { selectedUserId, todos, errorMessage } = this.state;
 
     return (
       <div className="App">
         <div className="App__sidebar">
-          <TodoList />
+          {todos.length > 0
```

Comment:
This would be enough
```suggestion
          {todos.length
```